presto-accumulo/src/main/java/com/facebook/presto/accumulo/AccumuloMetadata.java

```diff
@@ -162,7 +162,7 @@ public void dropView(ConnectorSession session, SchemaTableName viewName)
     public Map<SchemaTableName, ConnectorViewDefinition> getViews(ConnectorSession session, SchemaTablePrefix prefix)
     {
         ImmutableMap.Builder<SchemaTableName, ConnectorViewDefinition> builder = ImmutableMap.builder();
-        for (SchemaTableName stName : listViews(session, prefix.getSchemaName())) {
+        for (SchemaTableName stName : listViews(session, Optional.ofNullable(prefix.getSchemaName()))) {
```

Comment:
I think `SchemaTablePrefix` should also be changed in the same spirit.

Reply:
Sure, but let's do it in the separate PR.